src/Fieldset/README.md

```diff
@@ -0,0 +1,29 @@
+# Fieldset
+
+Wraps many inputs in a border and allows validating them as one form control.
+
+## API
+
+### `dangerIconDescription` (string or element; optional)
+`dangerIconDescription` can be used to provide a screen-reader description of the "exclamation" icon used in the validation message displayed when `isValid` is false and `themes` includes "danger". The default is an empty string.
+
+### `children` (string, element, or arrays of strings or elements; optional)
+`children` specifies the elements that the Fieldset will wrap. The default is an empty string.
+
+### `className` (string array; optional)
+`className` specifies Bootstrap class names to apply to the input component. The default is an empty array.
+
+### `id` (string; optional)
```

Comment:
If somebody wants to use an `id` of `1` will this result in a `PropType` error? Or will `1` get stringified in-time?

Reply:
I think that would result in a proptype error, but I can verify that.

Reply:
![screenshot-20180420143153-969x109](https://user-images.githubusercontent.com/1505923/39067696-9e463a38-44a7-11e8-9213-c00b9d9cd091.png)

Yup, PropType error.

Reply:
Why wouldn't an `id` value of `1` be valid?

Reply:
I think supporting both would unnecessarily complicate things because we later do string operations on the id because it is assumed to be a string. Also, I feel like an `id` attribute that is just a number is not very descriptive.